src/js/components/Form/stories/TriggerValidationUsingFormField.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import { StatusGood } from 'grommet-icons';
+import { Box, Button, Grommet, Form, FormField } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+export const TriggerValidationUsingFormField = () => (
+  <Grommet full theme={grommet}>
+    <Box fill align="center" justify="center">
+      <Box width="medium">
+        <Form
+          onReset={(event) => console.log(event)}
+          onSubmit={({ value }) => console.log('Submit', value)}
+        >
+          <FormField
+            label="First Name"
```

Comment:
It would help to add some text to this story or maybe to the field labels themselves that say what it is doing (ie validate on blur etc). Otherwise, it's hard to figure out what the example is trying to portray

Reply:
Sure, I will make the story descriptive.